CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [0.12.0]
 
 ### Added
-* Support pushing [`fenix`](https://github.com/mozilla-mobile/fenix) to Google Play 
+* Support pushing [`fenix`](https://github.com/mozilla-mobile/fenix) to Google Play
+
+### Changed
```

Comment:
Nit: https://github.com/mozilla-releng/pushapkscript/releases/tag/0.12.0 is already out. Let's put this section under 0.13.0